mods/seasonal/abilities.js

```diff
@@ -108,6 +108,11 @@ let BattleAbilities = {
 					this.boost({atk: -1}, target, pokemon, this.getAbility('intimidate'));
 				}
 			}
+			for (const action of this.queue) {
```

Comment:
I have no idea who gave the description, but Stimulated Pride is just swift swim + intimidate

Reply:
Yeah looks like lycan is correct on both counts.

Reply:
Meaning this needs to be removed.